src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -34,13 +38,31 @@ function ProviderSelectionField({
   );
   const loadingProviders =
     requestStatus === FETCH_STATUS.loading ||
-    requestStatus === FETCH_STATUS.notStarted;
+    requestStatus === FETCH_STATUS.notStarted ||
+    requestLocationStatus === FETCH_STATUS.loading;
```

Comment:
I think it probably makes sense to have both a `loadingProviders` and a `loadingLocation`, kept separate, since the loading messages would be different and you also might hide different parts of the page depending on which is loading (e.g. loading providers doesn't hide the residential address, but getting the current location would, based on the screens).

Reply:
Done.